### PR TITLE
Wc add show

### PR DESCRIPTION
**Pull request by wcampbell91**

### CODE REVIEW

**Review by jeaninedavis — APPROVED** [review]
it works!

File: src/javascripts/components/addSouv/addSouv.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const addSouvForm = (e) => {
                         <input type="text" class="form-control" id="addSouv-imageUrl">
                       </div>
                       <div class="form-check">
-                        <input type="checkbox" class="form-check-input" id="exampleCheck1">
+                        <input type="checkbox" class="form-check-input" id="addSouv-isAvailable">
                         <label class="form-check-label" for="exampleCheck1">Is Currently Available</label>
```
Comment (luketlancaster) [inline src/javascripts/components/addSouv/addSouv.js:27]: you should change the `for` to match the `id` that this label is...labeling

File: src/javascripts/components/showList/showList.js [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
-      let domString = `
-                    <div class="text-center">
-                      <h2 class="text-center">Upcoming Events</h2>
-                      <button class="btn btn-primary auth-button text-center" id="add-show">Add New Event</button>
-                    </div>
-                      <div class="display-flex flex-wrap myShows card-deck">`;
-      shows.forEach((show) => {
-        domString += showBuilder.showCardBuilder(show);
-      });
+  showBuilder.showCardBuilder();
```
Comment (luketlancaster) [inline src/javascripts/components/showList/showList.js:10]: trying to think of a world where you would want to do the data call in here, then pass in `show` to the `showCardBuilder` inside of a `shows.forEach`, but I'm coming up empty 🙃

**Review by luketlancaster — COMMENTED** [review]
just a couple of thoughts!

**Pull request merged**